PayKids/presentation/src/main/java/com/paykids/presentation/view/diary/AnalysisConsumeFragment.kt

```
@@ -89,9 +155,36 @@ class AnalysisConsumeFragment : BaseFragment<FragmentAnalysisConsumeBinding>() {
+
+        month += increment
+
+        if (month > 12) {
+            month = 1
+            year += 1
+        } else if (month < 1) {
+            month = 12
+            year -= 1
+        }
+
+        return String.format("%04d-%02d", year, month)
```

detekt (1.23.7)

[warning] String.format("%04d-%02d", year, month) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)